.github/workflows/workflow.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+FILE_PATH="$1"
```

Comment:
If it is not an environment variable, use lower_snake_case for the variable name.

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/1223/commits/5186ac984da5e69ccbe41d59f9ffb7fe80ede115